ksql-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -173,24 +173,31 @@ selectItem
 
 
 relation
-    : left=relation
-      ( CROSS JOIN right=aliasedRelation
-      | joinType JOIN rightRelation=relation joinCriteria
-      | NATURAL joinType JOIN right=aliasedRelation
-      )                                           #joinRelation
-    | aliasedRelation                             #relationDefault
+    : left=aliasedRelation joinType JOIN right=aliasedRelation joinCriteria joinWindow #joinRelation
+    | aliasedRelation #relationDefault
     ;
 
 joinType
     : INNER?
-    | LEFT OUTER?
-    | RIGHT OUTER?
-    | FULL OUTER?
+    | OUTER?
+    | LEFT?
```

Comment:
Change back to `LEFT OUTER?`.
Standard SQL syntax: https://www.w3schools.com/sql/sql_join_left.asp